package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "vnc-keymaps",
+  "version": "0.0.1",
+  "type": "module",
+  "description": "Keymps for VNC",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    "import": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js"
+    },
+    "require": {
+      "types": "./dist/index.d.cts",
+      "require": "./dist/index.cjs"
+    }
+  },
+  "files": [
+    "./dist"
+  ],
```

Comment:
Does this need to include package.json, and README.md?

Reply:
https://classic.yarnpkg.com/en/docs/package-json#toc-files doesn't say anything about always included files.  But if you tested it and everything looks good, then good!

Reply:
tested also with `yarn pack` -works the same as `npm pack`